src/styles/main.scss

```
@@ -1,3 +1,53 @@
 body {
   margin: 0;
+  font-family: Arial, serif;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, auto));
+}
+
+.container__blocks {
```

```suggestion
.container__block {
```
это класс одного блока